agents/healer.py

```
@@ -0,0 +1,114 @@
+        Alert Summary: {summary}
+        Alert Description: {description}
+
+        Based on this, suggest a file to modify and the code patch to apply.
+        """
+
+        generated_code = await self.kimi_client.generate_hotfix(prompt)
+
+        try:
+            blueprint = json.loads(generated_code)
+            return blueprint
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)